assets/src/scripts/wdfn-home/wdfn-param-filters/index.js

```diff
@@ -111,10 +112,45 @@ const WDFNParamFilters = (node, store) => {
   const setCount = (_, count) => {
       if (typeof count !== 'number') return 0;
       document.querySelector('#result-count span').textContent = count;
+      store.dispatch(applyLoadingState('loaded'));
+  };
+
+  const setLoadingState = (_, loadingState) => {
+    const mapContainerEl = document.getElementById('wdfn-map-container');
+    const submitBtn = document.getElementById('wdfn-search-submit');
+
+    mapContainerEl.dataset.loadingState = loadingState;
+    if (loadingState == 'loaded') submitBtn.innerText = 'Update results';
+  };
+
+  const toggleSearchEnabled = (_, filters) => {
+    let activeFilters = 0;
+    const filterInstructions = document.querySelector('.filter-instructions');
+
+    const submitBtn = document.getElementById('wdfn-search-submit');
```

Comment:
Another thing that I will need to think about going forward if we should think of these components as reusable in which case, using static IDs to select elements seems the wrong thing to do.